publiccode.yaml

```diff
@@ -0,0 +1,50 @@
+publiccodeYmlVersion: "0.2"
+name: Signalen frontend
+url: "https://github.com/signalen/frontend"
+softwareVersion: "1"
+releaseDate: "2021-05-06"
+platforms:
+  - web
+categories:
+  - it-development
+usedBy:
+  - Gemeente Amsterdam
+developmentStatus: development
+softwareType: standalone/web
+description:
+  nl:
+    localisedName: Signalen frontend
+    genericName: Signalen frontend
+    shortDescription: >-
+      This project provides a web frontend for Signalen, an application that helps cities manage and prioritize nuisance reports.
+    longDescription: >-
+      This project provides a web frontend for Signalen, an application that helps cities manage and prioritize nuisance reports.
+    documentation: "https://github.com/Signalen/frontend/blob/develop/README.md"
+    features:
+    - Meldingen classificeren
+    - Meldingen routeren
+    - Meldingen behandelen
+    - Rapporteren
+legal:
+  license: EUPL-1.2
+  mainCopyrightOwner: Signalen
+  repoOwner: Signalen
+maintenance:
+  type: external
+  contacts:
+    - name: Jacco Brouwer
+      email: jacco.brouwer@vng.nl
```

Comment:
Add a contact of Amsterdam as well? @aditudorache @MauritsR

Reply:
Do you by any chance have an email for @aditudorache @MauritsR?

Reply:
For now I would add the contacts Amy van Someren (Gemeente 's-Hertogenbosch) and René van Sprang (Gemeente Amsterdam). Their contacts details are here: https://signalen.org/contact/

Reply:
Thanks, ill make sure to add them.